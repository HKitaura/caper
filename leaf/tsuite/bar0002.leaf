require leaf;

fun main(): int
{
	if( true ) {
		1;
	} else {
		false;
	}
	0;
}
