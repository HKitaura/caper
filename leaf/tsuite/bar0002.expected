leaf: type mismatch: <int> at true-clause with <bool> at false-clause; file: tsuite/bar0002.leaf, line: 5, column: 5
